cerberus/inspect/inspect.py

```diff
@@ -18,3 +19,11 @@ def inspect_component(namespace):
         logging.info("Deleted existing %s directory" % (dir_name))
     command_out = runcommand.invoke("oc adm inspect ns/" + namespace + " --dest" "-dir=" + dir_name + " | tr -d '\n'")
     logging.info(command_out)
+
+
+def inspect_operator(operator):
+    dir_name = "inspect_data/" + operator + "-logs.out"
+    if os.path.isdir(dir_name):
+        runcommand.invoke("rm -R " + dir_name)
+        logging.info("Deleted existing %s directory" % (dir_name))
+    runcommand.invoke("kubectl describe co " + operator + " >> " + str(dir_name))
```

Comment:
Hitting 
```
2022-03-13 15:52:30,058 [INFO] Iteration 7: Failed operators
2022-03-13 15:52:30,058 [INFO] ['monitoring']

/bin/sh: inspect_data/monitoring-logs.out: No such file or directory
2022-03-13 15:52:30,068 [ERROR] Failed to run kubectl describe co monitoring >> inspect_data/monitoring-logs.out
2022-03-13 15:52:30,068 [ERROR] Error: Command 'kubectl describe co monitoring >> inspect_data/monitoring-logs.out' returned non-zero exit status 1.
2022-03-13 15:52:30,189 [INFO] []
```

Reply:
Think I've fixed this with using oc adm inspect with "--dest-dir"